tests/rptest/tests/delete_records_test.py

```diff
@@ -113,18 +115,22 @@ def _apply_local_retention_settings(self):
             f"Turning on aggressive local retention, log_segment_size: {self.log_segment_size} retention.local.target.bytes: {self.local_retention}"
         )
         self.rpk.alter_topic_config(
-            self.topic, TopicSpec.PROPERTY_RETENTION_LOCAL_TARGET_BYTES,
+            TEST_TOPIC_NAME, TopicSpec.PROPERTY_RETENTION_LOCAL_TARGET_BYTES,
```

Comment:
then you could use `self.topic` it would refer to the random name

Reply:
Please take a look at the latest commit. I'm want to use multiple topics in this test, hence the constant introduced here and others introduced later. Referring to topic names by something like `self.topics[1]` is too confusing in my opinion.